VK_App/VK_App/ViewControllers/FirstBarVC/FriendsPhotosViewController.swift

```diff
@@ -11,37 +11,79 @@ final class FriendsPhotosViewController: UIViewController {
 
     // MARK: - Private properties
 
-    private var photoNames: [String] = []
-    private var userImages: [UIImage] {
-        photoNames.compactMap { UIImage(named: $0) }
+    private let imageLoader = LoadingImage.shared
+    private let disQueue = DispatchQueue.global(qos: .userInteractive)
+    private var photoNames: [String] = [] {
+        didSet {
+            disQueue.sync {
```

Comment:
```suggestion
            DispatchQueue.main.async {
```
sync прекращает работу других потоков, зачем перекрывать все потоки по которым могут идти данные, чтоб просто загрузить фотки на экран?